examples/hybrid/sphere/deformation_flow.jl

```diff
@@ -43,6 +43,7 @@ const z_c = 5.0e3 # initial altitude of tracers
 const R_t = R / 2 # horizontal half-width of tracers
 const Z_t = 1000.0 # vertical half-width of tracers
 const κ₄ = 1.0e16 # hyperviscosity
+const C = 1.0 # flux-correction coefficient
```

Comment:
Yes, this corresponds exactly to the third-order case, and this is exactly what I wanted to test in this regression test. 
I tried several values, but since the third-order is better (in terms of dissipation) relative to its low-order counterparts, they all give a bit grosser errors. So I thought that for a regression test it is actually best to identically match the existing test with existing tolerances to show that this indeed coincides in that case with the existing case.